install_scripts/templates/swarm/init.sh

```diff
@@ -25,6 +25,7 @@ NO_CE_ON_EE="{{ no_ce_on_ee }}"
 HARD_FAIL_ON_LOOPBACK="{{ hard_fail_on_loopback }}"
 HARD_FAIL_ON_FIREWALLD="{{ hard_fail_on_firewalld }}"
 ADDITIONAL_NO_PROXY=
+REPLICATED_USERNAME=replicated
```

Comment:
why not like this? https://github.com/replicatedhq/replicated-installer/blob/76f9d7d3b9feb2395ec33f3d5de2b009eae2cf56/install_scripts/templates/replicated-operator.sh#L22

Reply:
how come you dont have to add the replicated user to the worker join ?

Reply:
Updated both. The missing `{{ replicated_username }}` would have broken old installs by trying to run them as root without the migrations in the daemon.